src/consensus/pbft/libbyz/certificate.h

```diff
@@ -374,6 +374,7 @@ bool Certificate<T>::add(T* msg)
     bmap.set(id);
     Message_val& val = vals[0];
     val.count++;
+    cur_size = 1;
```

Comment:
`cur_size` holds the `current number of elements in vals`. Needed here so that if the `Val_iter` is called [here](https://github.com/microsoft/CCF/blob/master/src/consensus/pbft/libbyz/certificate.h#L284) then the iterator will return this message, since it _is_ stored in the certificate